### PR TITLE
mprocs: init at 0.2.2

### PR DESCRIPTION
**Pull request by GaetanLepage**
###### Description of changes

mprocs runs multiple commands in parallel and shows output of each command separately.
Project homepage: https://github.com/pvolok/mprocs

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4519,6 +4519,12 @@
       fingerprint = "0200 3EF8 8D2B CF2D 8F00  FFDC BBB3 E40E 5379 7FD9";
     }];
   };
+  GaetanLepage = {
+    email = "gaetan@glepage.com";
+    github = "GaetanLepage";
+    githubId = 33058747;
+    name = "Gaetan Lepage";
+  };
```
Comment (Leixb) [inline maintainers/maintainer-list.nix:4527]: This should be in a separate commit named: `maintainers: add GaetanLepage`

File: pkgs/tools/misc/mprocs/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-9wac2jlh8MLggcYuEHVUPYPjhfzGN+4o1XT8B9pj4f8=";
+  };
+
+  cargoSha256 = "sha256-vGhoM5VMJO+ppLk8DetNZ8UteU6ifdtRe1HNJ0dSB+0=";
+
+  meta = with lib; {
+    description = "mprocs runs multiple commands in parallel and shows output of each command separately.";
+    homepage = "https://github.com/BurntSushi/ripgrep";
```
Comment (Leixb) [inline pkgs/tools/misc/mprocs/default.nix:18]:
```suggestion
    homepage = "https://github.com/pvolok/mprocs";
```

File: pkgs/tools/misc/mprocs/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mprocs";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "pvolok";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```
Comment (Leixb) [inline pkgs/tools/misc/mprocs/default.nix:10]: I don't think `refs/tags` is needed
```suggestion
    rev = "v${version}";
```

**Review by Leixb — CHANGES_REQUESTED** [review]
Some comments on issues that to follow the guidelines. (https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md)

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (GaetanLepage) [reply maintainers/maintainer-list.nix:4527]: How exactly should I address this ? Should I revert this change and make a new separate commit ? Wouldn't this pollute the history tree ?

Comment (Leixb) [reply maintainers/maintainer-list.nix:4527]: You should only have two commits, one for maintainer and the other for adding the package. To do so you can either use git rebase or git reset, make new commits and force push them.

**Comment by GaetanLepage** [comment]
@Leixb I have fixed the way I committed this PR.
Is there anything more for me to do to ease the work of the repo maintainers ?

**Review by Leixb — APPROVED** [review]
